Safety/Inc/imu.hpp

```diff
@@ -12,38 +11,50 @@
 /* Exported functions prototypes ---------------------------------------------*/
 // void Error_Handler();
 
-
 struct IMUData_t {
   double gyro_x, gyro_y, gyro_z; 
   double accel_x, accel_y, accel_z;
+  double mag_x, mag_y, mag_z;
 }; 
 
 class IMU {
   public: 
     virtual void GetResult(IMUData_t& Data) = 0; 
 }; 
 
-class MPU6050 : public IMU {
+class BMX160 : public IMU {
```

Comment:
I'm still not a fan of how the MPU6050 class was nuked in favour of the BMX160 class. I think both can live alongside each other in harmony. I would say we can add re-introduction of the MPU6050 driver into the asana backlog so that we can configure which IMU we want to use as appropriate.